src/components/cc-pricing-estimation/cc-pricing-estimation.js

```diff
@@ -26,457 +80,921 @@ const SIGN_UP_URL = 'https://api.clever-cloud.com/v2/sessions/signup';
+        </p>
+        <button class="header__toggle-btn" @click="${this._onToggle}">
+          ${this._isCollapsed ? i18n('cc-pricing-estimation.show') : i18n('cc-pricing-estimation.hide')}
+        </button>
       </div>
     `;
   }
 
-  _renderSmallSelectedPlans () {
+  _renderHeaderWithoutToggle () {
+    const productCount = this._getProductCount();
 
-    const selectedPlans = this.selectedPlans ?? [];
-
-    if (selectedPlans.length === 0) {
-      return html`
-        <div class="empty-text">
-          ${i18n('cc-pricing-estimation.empty-list')}
-        </div>
-      `;
-    }
+    return html`
+      <div class="header">
+        <strong class="header__heading">${i18n('cc-pricing-estimation.heading')}</strong>
+        <span class="header__badge">
+          ${productCount}
+          <span class="visually-hidden">${i18n('cc-pricing-estimation.count.label', { productCount })}</span>
+        </span>
+      </div>
+    `;
+  }
 
-    return selectedPlans.map((plan) => {
-
-      const { totalPriceDailyWithCode, totalPriceMonthlyWithCode } = this._getTotalPrices(plan);
-
-      return html`
-        <div class="plan">
-
-          <cc-button
-            class="delete-btn"
-            danger
-            outlined
-            image=${deleteSvg}
-            hide-text
-            circle
-            @cc-button:click=${() => this._onDeletePlan(plan)}
-          >
-            ${i18n('cc-pricing-estimation.delete')}
-          </cc-button>
-
-          <div class="product-name">${plan.productName}</div>
-
-          <cc-input-number
-            label=${i18n('cc-pricing-estimation.quantity')}
-            hidden-label
-            class="input-number"
-            value=${plan.quantity}
-            min="0"
-            controls
-            @cc-input-number:input=${(e) => this._onChangeQuantity(plan, e.detail)}
-          ></cc-input-number>
-
-          <div class="plan-name">
-            <span class="plan-name-label">${i18n('cc-pricing-estimation.plan')}</span>
-            <span class="plan-name-text">${plan.name}</span>
-          </div>
+  /**
+   * @param {Plan} plan - the plan to render
+   */
+  _renderSelectedPlan (plan) {
+    const totalPlanPrice = this._getTotalPlanPrice(plan);
+    const hasFeatures = Array.isArray(plan.features);
 
-          <div class="feature-list">
-            <div class="feature">
-              <div class="feature-name">${i18n('cc-pricing-estimation.price-name-daily')}</div>
-              <div class="feature-value">${i18n('cc-pricing-table.price', totalPriceDailyWithCode)}
-              </div>
-            </div>
-            <div class="feature">
-              <div class="feature-name">${i18n('cc-pricing-estimation.price-name-monthly')}</div>
-              <div class="feature-value">${i18n('cc-pricing-estimation.price', totalPriceMonthlyWithCode)}</div>
+    return html`
+      <div class="plan">
+        <details class="plan__toggle">
```

Comment:
praise: I really like this implementation! I'm wondering if we should use that for `<cc-expand>` component.

Reply:
@pdesoyres-cc You mean the `cc-block`, The `cc-expand` implements animation when something's height changed but it doesn't have a toggle mode.